lib/client/api.go

```diff
@@ -2402,112 +2402,69 @@ func PlayFile(ctx context.Context, filename, sid string, speed float64, skipIdle
 }
 
 // SFTP securely copies files between Nodes or SSH servers using SFTP
-func (tc *TeleportClient) SFTP(ctx context.Context, args []string, port int, opts sftp.Options, quiet bool) (err error) {
+func (tc *TeleportClient) SFTP(ctx context.Context, source, destination string, opts sftp.Options) (err error) {
 	ctx, span := tc.Tracer.Start(
 		ctx,
 		"teleportClient/SFTP",
 		oteltrace.WithSpanKind(oteltrace.SpanKindClient),
 	)
 	defer span.End()
 
-	if len(args) < 2 {
-		return trace.Errorf("local and remote destinations are required")
-	}
-	first := args[0]
-	last := args[len(args)-1]
+	localCopy := strings.ContainsRune(source, ':')
+	remoteCopy := strings.ContainsRune(destination, ':')
```

Comment:
In the previous version of this code, I understood what `isRemoteDest(first)` and `isRemoteDest(last)` were supposed to mean. However, I find it hard to understand why these variables are called `localCopy` and `remoteCopy`.